db/schema.rb

```diff
@@ -24,6 +24,7 @@
     t.string   "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "mood_id"
```

Comment:
i am not sure if we need this association. food has many moods through recommendations